bench.go

```diff
@@ -1,14 +1,40 @@
 package main
 
-import "time"
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/prometheus/client_golang/api"
+	v1 "github.com/prometheus/client_golang/api/prometheus/v1"
+	"github.com/prometheus/common/model"
+)
 
 type bench interface {
 	run() error
 	collect() error
 }
 
+type timePoint struct {
+	addTime     time.Time
+	balanceTime time.Time
+}
+
+type stats struct {
```

Comment:
will we collect the balance region operator?

Reply:
Collect it by count of schedules in balance region scheduler.